test/instance.ts

```diff
@@ -263,6 +256,23 @@ describe('Bigtable/Instance', () => {
         };
         instance.createAppProfile(APP_PROFILE_ID, options, assert.ifError);
       });
+
+      it('a set of clusters', done => {
```

Comment:
nit - update this description

Reply:
I updated this to say `a set of cluster objects`, but I think you might have wanted something else. In this describe block our description string says, `"should respect the routing option with"` and then each `it` describes what we pass into the routing option. In this case, on the line that says, `const options = {routing: new Set(clusters)};` we are setting the routing option to a set of clusters.